create_release.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env sh
+
+# Ensure that exactly one argument is provided
+if [ "$#" -ne 1 ]; then
+    echo "Usage: $0 {major|minor|patch}"
+    exit 1
+fi
+
```

Comment:
Another best practice is to switch to the right working directory:

```suggestion
cd "$(dirname "$0")"
```